core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/user/dataset/type/DatasetFileNode.scala

```diff
@@ -103,7 +103,7 @@ object DatasetFileNode {
       val fileType =
         if (Files.isDirectory(currentPhysicalNode.getAbsolutePath)) "directory" else "file"
```

Comment:
Don't you also have to fix this if condition?

Reply:
Good catch! Fixed it.